[Build] Philosophy page sample test run fails

**Description:**

Philosophy page code samples test run fails on windows

```shell
Error: Could not find or load main class org.ballerinalang.launcher.Main
   Building annotations.bal
Error: Could not find or load main class org.ballerinalang.launcher.Main
   Building api-constructs.bal
[ERROR] "annotations.bal" failed with exit code 1.
Error: Could not find or load main class org.ballerinalang.launcher.Main
[ERROR] "api-constructs.bal" failed with exit code 1.
   Building asynchronous.bal
Error: Could not find or load main class org.ballerinalang.launcher.Main
[ERROR] "asynchronous.bal" failed with exit code 1.
   Building circuit-breaker.bal
Error: Could not find or load main class org.ballerinalang.launcher.Main
[ERROR] "circuit-breaker.bal" failed with exit code 1.
   Building distributed-transactions.bal
Error: Could not find or load main class org.ballerinalang.launcher.Main
   Building docker-and-kubernetes.bal
[ERROR] "distributed-transactions.bal" failed with exit code 1.
Error: Could not find or load main class org.ballerinalang.launcher.Main
[ERROR] "docker-and-kubernetes.bal" failed with exit code 1.
   Building json-and-xml.bal
Error: Could not find or load main class org.ballerinalang.launcher.Main
[ERROR] "json-and-xml.bal" failed with exit code 1.
   Building logic-language.bal
```

**Affected Product Version:**
v0.9.046

**OS, DB, other environment details and versions:**    
Windows 10

**Steps to reproduce:**
Run `mvn install` on `~/website` folder

--- a/website/tools/build-philosophy-samples.sh
+++ b/website/tools/build-philosophy-samples.sh
@@ -7,7 +7,14 @@ echo "....Building philosophy samples..."
 
 if [ -z "$BALLERINACMD" ] ; then
   if [ -n "$BALLERINA_HOME"  ] ; then
-    BALLERINACMD="$BALLERINA_HOME/bin/ballerina"
+	if [ $OSTYPE == "msys" ];
+	  EXEC=""
+	  then
+		EXEC="bin/ballerina.bat"
+	  else
+        EXEC="bin/ballerina"
+	fi
+	BALLERINACMD=$BALLERINA_HOME$EXEC 
   else
     BALLERINACMD=ballerina
   fi
@@ -18,14 +25,13 @@ if [ -z "$BALLERINA_HOME" ]; then
   exit 1
 fi
 
-if [ ! -x "$BALLERINACMD" ] ; then
+if [ ! -x "$BALLERINACMD" && $OSTYPE -ne "msys" ] ; then
   echo "[ERROR] BALLERINA_HOME is not defined correctly."
   echo "[ERROR] cannot execute $BALLERINACMD"
   exit 1
 fi
 
-
-echo "Ballerina Home :" $BALLERINA_HOME
+echo "Ballerina Home : $BALLERINA_HOME"
 $BALLERINACMD -v
 
 exit_on_error() {

